publishing-api/config.ru

```diff
@@ -0,0 +1,14 @@
+require 'sidekiq'
+
+Sidekiq.configure_client do |config|
+  config.redis = {
+    host: ENV["REDIS_HOST"] || "127.0.0.1",
+    port: ENV["REDIS_PORT"] || 6379,
+    namespace: "publishing_api",
+  }
+end
+
+require 'sidekiq/web'
+map '/publishing-api' do
+  run Sidekiq::Web
+end
```

Comment:
Not sure how that would work; the Redis configuration is using publishing_api as a namespace, as defined in the app itself. But the path is publishing-api, consistent with our convention for paths generally, which use dashes rather than underscores.